### DISCUSSION TITLE
Tutorial files unavailable

### DISCUSSION THREAD

**Discussion by scgmelo** [hahnlab/CAFE5#195] [Tutorial] [answered]
I tried to access the tutorial files in https://iu.box.com/v/cafetutorial-files but they are unavailable. My question is about the command:

python mcl2rawcafe.py -i dump.blast_output.mci.I30 -o unfiltered_cafe_input.txt -sp "ENSG00 ENSPTR ENSPPY ENSPAN ENSNLE ENSMMU ENSCJA ENSRNO ENSMUS ENSFCA ENSECA ENSBTA"

There is a right order to write the sp? I'm using fasta file from NCBI and when I get to the blast_output.abc there is nothing but the protein ID form subject and query, so I don't know how the sp are ordenated and when I do this command I have a blank unfiltered_cafe_input.txt. Oder question is should I use evalue o similarity for the blast_output.abc file?

**Comment by benfulton** [comment, marked as answer]
Hi,

The tutorial files have moved to https://datacore.iu.edu/concern/data_sets/6m311q02z?locale=en . I see that the old directory is still referenced in tutorial.md so I will update that in Github at least.

**Reply by scgmelo** [reply]
Thank you!!
